src/rwr.py

```diff
@@ -0,0 +1,155 @@
+import warnings
+from pathlib import Path
+
+import pandas as pd
+
+from src.prm import PRM
+from src.util import add_rank_column, prepare_volume, run_container
+
+__all__ = ['RandomWalk']
+
+class RWR(PRM):
+    # we need edges (weighted), source set (with prizes), and target set (with prizes).
+    required_inputs = ['edges', 'sources', 'targets']
+
+    @staticmethod
+    def generate_inputs(data, filename_map):
+        """
+        Access fields from the dataset and write the required input files
+        @param data: dataset
+        @param filename_map: a dict mapping file types in the required_inputs to the filename for that type
+        @return:
```

Comment:
empty return comment

Reply:
I left this empty because all the other algorithms implemented in SPRAS also have an empty return comment. I would appreciate your recommendation on what should be included here. If you have any suggestions, I can update the return comment for this algorithm as well as for the other algorithms in SPRAS.

Reply:
Because `generate_inputs` doesn't return anything I suggest deleting the `@return:` line from the docstring. We'll have to formalize this once we finally start automatically generating documentation from docstrings with something Sphinx.